jscomp/syntax/tests/parsing/recovery/pattern/expected/list.res.txt

```diff
@@ -24,16 +24,17 @@
 
 
   Syntax error!
-  tests/parsing/recovery/pattern/list.res:4:13
+  tests/parsing/recovery/pattern/list.res:4:9-11
 
   2 │ | list{} => ()
   3 │ | list{1, list{} => ()
   4 │ | list{}...1, ...list{3, 4} => ()
   5 │ }
   6 │ 
 
-  I'm not sure what to parse here when looking at ",".
+  Did you forget a `}` here?
```

Comment:
to be honest, the former error message was not great, the newer one is not better, but I'm not even sure what should be the error message here.